Buildings/Fluid/Humidifiers/EvaporativeCoolers/Baseclasses/Validation/DirectCalculations.mo

```diff
@@ -0,0 +1,180 @@
+within Buildings.Fluid.Humidifiers.EvaporativeCoolers.Baseclasses.Validation;
+model DirectCalculations
+  "Validation of the DirectCalculations block"
+  extends Modelica.Icons.Example;
+
+  parameter Modelica.Units.SI.Area padAre = 0.6
+    "Area of the rigid media evaporative pad";
+  parameter Modelica.Units.SI.Length dep = 0.2
+    "Depth of the rigid media evaporative pad";
+  parameter Modelica.Units.SI.Length pAtm = 101325
+    "Atmospheric pressure";
+  parameter Modelica.Units.SI.ThermodynamicTemperature TDryBulSup_nominal = 296.15
+    "Nominal supply air drybulb temperature";
+  parameter Modelica.Units.SI.ThermodynamicTemperature TWetBulSup_nominal = 289.3
+    "Nominal supply air wetbulb temperature";
+  parameter Modelica.Units.SI.VolumeFlowRate V_flow_nominal = 1
+    "Nominal supply air volume flowrate";
```

Comment:
May not need to use the parameters. We can directly set the numbers to the `Modelica.Blocks.Sources.Constant` instances.

Reply:
We would like to retain the parameters since they are being used as inputs on various signal source types like constant and ramp. Will be easier to change these values through parameters on all instances